src/main/java/core/mate/academy/service/MachineProducer.java

```diff
@@ -1,11 +1,12 @@
 package core.mate.academy.service;
 
+import core.mate.academy.model.Machine;
 import java.util.List;
 
-public interface MachineProducer {
+public interface MachineProducer<T extends Machine> {
     /**
      * In your implementations - create 2-3 models, add them to the list and return
      * @return - the list of models
      */
-    List<Object> get();
+    List<T> get();
```

Comment:
The method `get` is defined to return `List<? extends Machine>`, but since the interface is generic with `T extends Machine`, it would be more consistent to return `List<T>`. This change would ensure that the method returns a list of the specific type `T` that extends `Machine`, aligning with the generic type parameter.